webcompat/static/css/development/components/navbar.css

```diff
@@ -53,7 +53,23 @@
     /*icon */
     .wc-Navbar-link-icon {
       font-size:1.2em;
+      display: inline-block;
+      background-repeat: no-repeat;
+      background-position: center center;
+      position: relative;
     }
+
+  /* "Contribute" link icon */
+  .wc-Icon--contribute {
+    background-image: url(/img/contribute.svg);
+    height: 1.2em;
```

Comment:
These height and width styles should move to line 55, `.wc-Navbar-link-icon` once the other icons are [changed to svg](https://github.com/webcompat/webcompat.com/issues/899).